roles/elk/tasks/main.yml

```diff
@@ -17,6 +21,12 @@
     dest: "{{ kibana_configuration }}"
   notify: restart kibana
 
+- name: configure oauth2_proxy
+  template:
+    src: oauth2_proxy.conf
+    dest: "{{ oauth2_proxy_configuration }}"
```

Comment:
Should the permissions or owner for this file be restricted? It looks like below the Github OAuth secret is in this file.